### PR TITLE
Add USA eco legions

### PR DESCRIPTION
**Pull request by henrykironde**
Level III and IV Ecoregions of the Continental
United States

### CODE REVIEW

File: scripts/us_eco_level_four.json [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    "Natural Resources", "spatial-data"
+    ],
+    "extent": {
+    "xMax": 2258224.8307000003,
+    "xMin": -2356069.457800001,
+    "yMax": 3172577.0456,
+    "yMin": 272048.49569999985
+    },
+    "version": "1.0.0",
+    "retriever_minimum_version": "2.1.0",
```
Comment (ethanwhite) [inline scripts/us_eco_level_four.json]: minimum version should be 2.2.0 right?

**Comment by henrykironde** [comment]
I separated them into different scripts because they have different citations for the levels.
But will add them and try to figure out the citation.
@ethanwhite in case you know the correct citation to the combined data help me and paste it here.

**Comment by ethanwhite** [comment]
Ah, that makes sense. I still think it's more useful from the user perspective to package all of these together. I think the correct long-term solution is to allow `citation` to be a list and present multiple citations if there are more than 1. It's not uncommon for datasets to ask for multiple papers to be cited so it's a general thing to improve. For now. I think we could probably just include both of them in a single string back to back.

**Comment by ethanwhite** [comment]
Once you've added the other citation back in let me know and I'll get this merged. I'll also open an issue about multiple citations.

**Comment by henrykironde** [comment]
we have decided to use "table_citations": {} for multi citations.

**Pull request merged**